client/mm/mm_arb_market_maker.go

```diff
@@ -529,7 +529,10 @@ func (a *arbMarketMaker) botLoop(ctx context.Context) (*sync.WaitGroup, error) {
 		defer bookFeed.Close()
 		for {
 			select {
-			case ni := <-bookFeed.Next():
+			case ni, ok := <-bookFeed.Next():
+				if !ok {
+					return
+				}
```

Comment:
This is good, but we should also shut down the bot completely. I think like this the bot will still appear to be running, but it will take no actions.

Reply:
We could also try to set everything up again with a new feed, but I don't think this will hit often so shutdown is probably cleanest.